gnd/src/main/java/com/google/android/gnd/repository/local/IntEnum.java

```diff
@@ -29,12 +28,12 @@
   int intValue();
 
   static <E extends IntEnum> int toInt(E enumValue, @NonNull E defaultValue) {
-    return enumValue == null ? defaultValue.intValue() : enumValue.intValue();
+    return enumValue == null ? defaultValue.intValue() : enumValue.intValue() ;
   }
 
   @NonNull
   static <E extends Enum<E> & IntEnum> E fromInt(
-      @NonNull E[] values, int intValue, @NonNull E defaultValue) {
-    return stream(values).filter(s -> s.intValue() == intValue).findFirst().orElse(defaultValue);
+      @NonNull Stream<E> values, int intValue, @NonNull E defaultValue) {
```

Comment:
It looks like maybe something leaked in here to revert this to taking a `Stream` argument -- not sure where that came from.

Reply:
Looks like I accidentally reverted when trying to do something clever with the PRs. Reverted the revert.